radixdlt-core/radixdlt/src/main/java/com/radixdlt/consensus/UnverifiedVertex.java

```diff
@@ -155,7 +163,7 @@ public static UnverifiedVertex create(
 		List<Txn> txns,
 		BFTNode proposer
 	) {
-		Objects.requireNonNull(qc);
+		requireNonNull(qc);
 
 		if (view.number() == 0) {
```

Comment:
Should be moved to view constructor.

Reply:
I don't think so. We don't allow it here, but genesis view is 0 and this is valid case.

Reply:
makes sense